test/specs/repo/extensions/nuke.js

```diff
@@ -46,27 +46,15 @@ describe( 'Nuke', function () {
 		await submitButtonEl.waitForDisplayed();
 		await submitButtonEl.click();
 		await browser.acceptAlert();
-	} );
-
-	it( 'Should delete the page in a job', async () => {
-		let result;
 
-		await browser.waitUntil(
-			async () => {
-				result = await browser.makeRequest(
-					process.env.MW_SERVER + '/wiki/Vandalism',
-					{ validateStatus: false },
-					{}
-				);
+		await browser.waitForJobs();
 
-				return result.status === 404;
-			},
-			{
-				timeout: 10000,
-				timeoutMsg: 'Page should be deleted by now.'
-			}
+		result = await browser.makeRequest(
+			process.env.MW_SERVER + '/wiki/Vandalism',
+			{ validateStatus: false },
+			{}
 		);
 
-		assert.strictEqual( result.status, 404 );
+		return result.status === 404;
```

Comment:
Please assert rather than return, for consistency's sake

Reply:
good catch, actually a copy pasta error. thanks. done